packages/nimble-components/src/table-column/date-text/tests/table-column-date-text-matrix.stories.ts

```diff
@@ -0,0 +1,61 @@
+import type { StoryFn, Meta } from '@storybook/html';
+import { html, ViewTemplate } from '@microsoft/fast-element';
+import { createMatrixThemeStory } from '../../../utilities/tests/storybook';
+import {
+    createMatrix,
+    sharedMatrixParameters
+} from '../../../utilities/tests/matrix';
+import { tableColumnDateTextTag } from '..';
+import { iconUserTag } from '../../../icons/user';
+import { Table, tableTag } from '../../../table';
+
+const metadata: Meta = {
+    title: 'Tests/Table Column Types',
+    parameters: {
+        ...sharedMatrixParameters()
+    }
+};
+
+export default metadata;
+
+const data = [
+    {
+        id: '0',
+        date: new Date('Dec 21, 2020, 3:45:03 PM').valueOf()
+    },
+    {
+        id: '1',
+        date: new Date('Dec 21, 2020, 3:45:03 PM').valueOf()
+    },
+    {
+        id: '2'
+    }
+] as const;
+
+// prettier-ignore
+const component = (): ViewTemplate => html`
+    <label style="color: var(--ni-nimble-control-label-font-color); font: var(--ni-nimble-control-label-font)">Date Text Table Column</label>
```

Comment:
Don't use the bare tokens: https://github.com/ni/nimble/blob/main/packages/nimble-components/docs/css-guidelines.md#design-tokens

If that pattern was copied from somewhere clean it up.

Reply:
It was copied from the anchor column matrix stories, but I'm cleaning that up in my PR removing placeholder support.